worlds/akasha.world/public/index.dev.html

```diff
@@ -5,7 +5,7 @@
   <meta charset="utf-8" />
   <meta
     http-equiv="Content-Security-Policy"
-    content="default-src 'self' 'unsafe-inline' 'unsafe-eval' cloudflare.com *.cloudflare.com *.ipfs.cf-ipfs.com *.cf-ipfs.com cloudflare-ipfs.com unpkg.com *.akasha.network *.akasha.world *.walletconnect.org *.infura.io *.ipfs.infura-ipfs.io *.ipfs.hub.textile.io http://localhost:* *.litgateway.com:* https://* wss://* wss://localhost:* data: ; img-src https: data: blob:"
+    content="default-src 'self' 'unsafe-inline' 'unsafe-eval' cloudflare.com *.cloudflare.com *.ipfs.cf-ipfs.com *.cf-ipfs.com cloudflare-ipfs.com unpkg.com *.akasha.network *.akasha.world *.walletconnect.org *.infura.io *.ipfs.infura-ipfs.io *.ipfs.hub.textile.io *.litgateway.com:* https://* https://localhost:* wss://localhost:* wss://* data: ; img-src https: data: blob:"
```

Comment:
Is `https://*` intended ?

Reply:
It was already there, didn't added in this pr